app/wrappers/wakes/google_analytics_api_wrapper.rb

```diff
@@ -31,6 +65,66 @@ def credentials
     @credentials ||= Google::Auth::ServiceAccountCredentials.from_env(AUTHORIZATION_SCOPE)
   end
 
+  Page = Struct.new(:rows, :end?)
```

Comment:
@benhutton Technically, it's `rows_of_page_views`. I don't think that's necessary, though.